packages/mwp-api-state/src/sync/index.js

```diff
@@ -134,70 +133,84 @@ export const apiRequestToApiReq = action =>
  * - API_ERROR  // deprecated
  * - API_COMPLETE
  */
-export const getFetchQueriesEpic = fetchQueriesFn => {
-	let locationIndex = 0; // keep track of location changes
+export const getFetchQueriesEpic = (resolveRoutes, fetchQueriesFn) => {
+	// keep track of location changes - will first be set by SERVER_RENDER
+	let currentLocation = {};
```

Comment:
instead of keeping track of an incrementing integer, we keep track of a `location` object - this will update any time `LOCATION_CHANGE` is fired, and we can use the value with `resolveRoutes` to find the matching route, from which we can construct the `/mu_api/` path